tools/interop-node/server/server.go

```diff
@@ -36,13 +37,31 @@ type Server struct {
 
 // NewServer creates a new interop server
 func NewServer(
-	config *config.Config,
+	config *cfg.Config,
 	ctx context.Context,
 	logger log.Logger,
 ) (*Server, error) {
 	logger = logger.With("server", "interop-node")
 
-	sc, err := client.NewSettlusClient(config, ctx, logger)
+	s := signer.NewSigner(ctx, config)
+	switch config.Feeder.SignerMode {
```

Comment:
Shouldn't calling `s.PubKey()` be enough regardless of the signer type? Seems like switch can be removed.